Source/DataSources/RectangleGeometryUpdater.js

```diff
@@ -208,9 +206,14 @@ define([
         options.granularity = defined(granularity) ? granularity.getValue(Iso8601.MINIMUM_VALUE) : undefined;
         options.stRotation = defined(stRotation) ? stRotation.getValue(Iso8601.MINIMUM_VALUE) : undefined;
         options.rotation = defined(rotation) ? rotation.getValue(Iso8601.MINIMUM_VALUE) : undefined;
-        options.closeBottom = defined(closeBottom) ? closeBottom.getValue(Iso8601.MINIMUM_VALUE) : undefined;
-        options.closeTop = defined(closeTop) ? closeTop.getValue(Iso8601.MINIMUM_VALUE) : undefined;
-        this._isClosed = defined(extrudedHeight) && defined(options.closeTop) && defined(options.closeBottom) && options.closeTop && options.closeBottom;
+        options.closeBottom = Property.getValueOrDefault(rectangle.closeBottom, Iso8601.MINIMUM_VALUE, true);
```

Comment:
I'm not crazy about defaulting values here, simply because I think defaults should be managed at the geometry level, not the updater level.  If a geometry default changes, these will get out of sync.  IT also ends up allocating memory to hold these variables when we really don't need when undefined will do.  That being said, I'm probably a little over concerned here and I'm okay with this change.  But just remember to take things like per-entity-property overhead into account when working at the DataSource level, because when you have 200,000 properties in the system, it can be the death of a thousand paper cuts.

Reply:
I made this change because apparently `true && undefined` is `undefined`.  So in the `isClosed` check I would have to do 
```js
isExtruded && extrudedHeight !== height && (!defined(options.closeTop) || options.closeTop === true) && (!defined(options.closeBottom) || options.closeBottom)
```

This change seemed better

Reply:
This doesn't matter anymore because of #6268.  (Also, `PolygonGeometryUpdater` already does this.  Write up an issue if you think it needs to be changed)